.github/workflows/release-create-branch.yml

```diff
@@ -0,0 +1,36 @@
+name: Release - Create Branch
+
+on:
+  workflow_dispatch:
+    inputs:
+      gitRef:
+        description: "Git Ref (e.g. main-456)"
+        required: true
+        default: ""
+      releaseType:
+        description: "Release type"
+        type: choice
+        required: false
+        options:
+          - release
+          - hotfix
+      versionName:
+        description: "Version name (must follow e.g. v1.0.0)"
+        required: true
+        default: "v0.0.0"
+
+jobs:
+  createBranch:
+    name: "Create ${{ inputs.releaseType }} branch"
+    runs-on: ubuntu-latest
```

Comment:
Believe that the EOL with ubuntu will be LF when the code is checked out and pushed to new branch. Just wanted to mention.

Reply:
Sorry, I am not following, why it matters? We are not changing any file here also 😉

Reply:
Oh yes. Never mind.